policyengine_us/variables/gov/states/co/ccap/co_ccap_parent_fee.py

```diff
@@ -0,0 +1,97 @@
+from policyengine_us.model_api import *
+
+
+class co_ccap_parent_fee(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Colorado Child Care Assistance Program parent fee"
+    reference = (
+        "https://docs.google.com/spreadsheets/d/1EEc3z8Iwu_KRTlBtd2NssDDEx_FITqVq/edit#gid=468321263",
+        "https://docs.google.com/spreadsheets/d/1HtPiC2qxclzWfBa7LRo2Uohrg-RCBkyZ/edit#gid=582762342",
+        "https://www.sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=11042&fileName=8%20CCR%201403-1#page=41",
+        "https://www.sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=11042&fileName=8%20CCR%201403-1#page=62",
+    )
+    unit = USD
+    definition_period = MONTH
+    defined_for = "co_ccap_eligible"
+
+    def formula(tax_unit, period, parameters):
+        year = period.start.year
+        if period.start.month >= 10:
+            instant_str = f"{year}-10-01"
+        else:
+            instant_str = f"{year - 1}-10-01"
+        p = parameters(instant_str).gov.states.co.ccap
+        # Calculate base parent fee and add on parent fee.
+        agi = tax_unit("adjusted_gross_income", period.this_year)
```

Comment:
Where does it say that IRS AGI determines the parent fee? I don't see that in any reference, and the spreadsheet describes a different more expansion income definition

Reply:
You are right, I made a mistake. It should be gross income in stead of AGI